serverless.yml

```diff
@@ -0,0 +1,27 @@
+service: app
+
+provider:
+    name: aws
+    region: us-east-1
+    runtime: provided.al2
+
+plugins:
+    - ./vendor/bref/bref
+
+functions:
+    default:
+        handler: index.php
+        name: task-${sls:stage}
+        layers:
+            - ${bref:layer.php-81}
```

Comment:
DISCUSS: Are we happy using their layers over building our own?

For the record, I think I probably am more confident in that now than I was but I think it's worth discussing.

Reply:
Yeah, we based our layers on the Bref layer anyway. I don't remember why we exactly did that, but now I see that our runtime just includes Composer and AWS tools which actually was a mistake - we need them only in dev env.

Reply:
I did it because I didn't want to base production services on unproven 3rd party stuff.  It's one thing to use the open source Bref code.  It's quite another to use a hosted 3rd party layer.

That said, we're 3 years on and I have rather more faith now than I did back then.